stdlib/bigarray.mli

```diff
@@ -61,6 +61,45 @@
      and {!Stdlib.input_value}).
 *)
 
+(** {1 Indexing operators}
+
+   Similarly to ordinary arrays, the elements of a big array, [a], can be
```

Comment:
I think we should be consistent about referring to it as a `Bigarray` rather than a `bigarray` or a `big array`.

```suggestion
   Similarly to ordinary arrays, the elements of a Bigarray, [a], can be
```

Reply:
This seems like a good change.

Reply:
Isn't "Bigarray" a little too grammatically alien as a noun? I have the impression that `bigarray` would mesh better. Am I mistaken? Moreover, the current documentation in `Bigarray.mli` uses "big array", so it may make sense to split this change in another PR.

Reply:
So the module is `Bigarray`, and capitalizing it and fusing the words makes it clear to the reader that you're looking at a term of art and not just some way of saying "if an array is large". At least, so it seems to me. @avsm further comment from your viewpoint?

Reply:
I think it should be "Bigarray"; this isn't about any old "big" array, but rather about a specific named thing.  As such a proper noun is appropriate.

Reply:
(It should maybe be in typewriter font.)

Reply:
`Bigarray` seems quite consensual. I will make the change on a separate PR.

Reply:
This PR have been updated to use `Bigarray` after #2208 .